components/Banner/examples/tests/snapshot/MDCBannerSnapshotTests.m

```diff
@@ -38,7 +39,11 @@ - (void)setUp {
   // test you wish to recreate the golden for).
   //  self.recordMode = YES;
 
-  self.bannerView = [[MDCBannerView alloc] initWithFrame:CGRectMake(0, 0, 350, 51)];
+  self.bannerView = [[MDCBannerView alloc] initWithFrame:CGRectZero];
+  UIEdgeInsets margins = UIEdgeInsetsZero;
+  margins.left = kBannerContentPadding;
+  margins.right = kBannerContentPadding;
+  self.bannerView.layoutMargins = margins;
```

Comment:
Please set `directionalLayoutMargins` on iOS 11+.

Reply:
Addressed in 0306a767c94b9d09ea2ecae308fe68d7417b5289.

Does Snapshot test also run on iOS 11+ above now?

Reply:
Our snapshot tests have always run on iOS 11.2